differ/differ.go

```diff
@@ -439,10 +439,13 @@ func generateWeeklyNotificationMessage(advisorURI string, accountID string, dige
 }
 
 // appendEventToNotificationMessage function adds a new event to the given notification message after constructing the payload string
-func appendEventToNotificationMessage(ruleURI string, notification *types.NotificationMessage, ruleName string, totalRisk int, publishDate string) {
+func appendEventToNotificationMessage(ruleURI string, notification *types.NotificationMessage, ruleName string, totalRisk int, publishDate string, clusterID string, module string, errorKey string) {
+	parsedModule := strings.Replace(module, ".", "|", -1)
```

Comment:
please move the string replacement process into another function, something like `generateNotificationPayloadURL`, so we don't have 8 parameters in the `appendEventToNotificationMessage function

Reply:
Done!